src/main/java/lt/vu/web/api/v1/helper/TimeManager.java

```diff
@@ -0,0 +1,48 @@
+package lt.vu.web.api.v1.helper;
+
+import javax.enterprise.context.RequestScoped;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+import java.util.Date;
+
+@RequestScoped
+public class TimeManager implements Serializable {
```

Comment:
I don't think this is necessary 😕 Why go through such hassle when we already have `Date` object?